runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/ClueScrollPlugin.java

```diff
@@ -230,17 +228,7 @@ public void onGameTick(final GameTick event)
 
 		if (clue instanceof EmoteClue)
 		{
-			equippedItems = new HashSet<>();
-
-			Item[] result = queryRunner.runQuery(new InventoryItemQuery(InventoryID.EQUIPMENT));
-
-			if (result != null)
-			{
-				for (Item item : result)
-				{
-					equippedItems.add(item.getId());
-				}
-			}
+			equippedItems = queryRunner.runQuery(new InventoryItemQuery(InventoryID.EQUIPMENT));
```

Comment:
Hmm, probably better would be to use client.getInvetory(InventoryID.EQUIPMENT)) to avoid query